person_class.rb

```diff
@@ -0,0 +1,19 @@
+class Person
+  def initialize(age, name = 'Unknown', parent_permission: true)
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+  end
+
+  def of_age?
+    @age >= 18
+  end
```

Comment:
Kindly implement few features that are missing from this class as stated in the requirement: 👇 

1.  GETTERS AND SETTERS methods are not yet created for `@id, @name, and @age and setters for @name and @age`. You can do this by `using atrr_reader and attr_accessor`. ℹ️ 
2. `@id` that should be generated randomly is not yet implemented.
3. `def of_age?` is expected to be a private method.